objects/conversion.go

```diff
@@ -138,3 +139,118 @@ func ToByteSlice(o Object) (v []byte, ok bool) {
 	//ok = false
 	return
 }
+
+// ToMap will try to convert object o to map[string]interface value.
+func ToMap(o Object) (v map[string]interface{}, ok bool) {
+	switch o := o.(type) {
+	case *Map:
+		for key, value := range o.Value {
+			v[key] = objectToInterface(value)
+		}
+		ok = true
+	}
+
+	//ok = false
+	return
+}
+
+// ToSlice will try to convert object o to []interface{} value.
+func ToSlice(o Object) (v []interface{}, ok bool) {
```

Comment:
Let's remove it if not used.

Reply:
Removed this whole commit, includes ToSlice, ToMap and objectToInterface. The latter was only being used by ToSlice and ToMap